setup.py

```diff
@@ -101,25 +94,24 @@ def run_tests(self):
 ]
 
 lint_reqs = [
-    "black>=22.3.0",
-    "flake8>=4.0.1",
+    "black>=23.7.0",
+    "flake8>=6.0.0",
     "flake8-absolute-import>=1.0",
-    "flake8-docstrings>=1.6.0",
-    "flake8-rst-docstrings>=0.2.5",
-    "flake8-implicit-str-concat==0.3.0",
-    "isort>=5.10.1",
-    "pre-commit>=2.19.0",
+    "flake8-docstrings>=1.7.0",
+    "flake8-rst-docstrings>=0.3",
+    "flake8-implicit-str-concat==0.4.0",
+    "isort>=5.12",
+    "pre-commit>=3.3",
 ]
 
 extra_reqs = {
-    "optional": optional_reqs,
     "doc": doc_reqs,
     "test": test_reqs,
     "lint": lint_reqs,
 }
 
 # Load the README.
-readme = Path("README.rst").read_text()
+readme = Path("README.md").read_text()
```

Comment:
this change make mandatory `long_description_content_type='text/markdown'` in the `setup()` function. 

https://packaging.python.org/en/latest/guides/making-a-pypi-friendly-readme/#including-your-readme-in-your-package-s-metadata

Reply:
:+1: